packages/app/src/runner/SpecRunnerContainerOpenMode.vue

```diff
@@ -30,13 +29,21 @@ const specs = computed(() => {
 
 watchSpec(specs)
 
-watchEffect(() => {
-  const specPath = router.currentRoute.value.query.file
+specStore.$subscribe((mutation, state) => {
```

Comment:
@lmiller1990 what do you think of using Pinia's `$subscribe` here? We aren't doing it elsewhere yet but it seems a bit more explicit precise than the`watchEffect`.

Reply:
Sure, it's just `watchEffect` under the hood. I still think we need to audit how the state and updates are flowing - it's starting to get a bit confusing, things are all over the place. I think we probably need a `useCurrentSpec` composable to group it all together.